src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -584,17 +584,20 @@ private void handlePotentialStuckRepairs(LazyInitializer<Set<String>> busyHosts,
     return () -> {
       LOG.debug("getMetricsForHost {} / {} / {}", node, localDc, nodeDc);
 
-      if (DatacenterAvailability.ALL != context.config.getDatacenterAvailability() && !nodeDc.equals(localDc)) {
+      if (DatacenterAvailability.ALL != context.config.getDatacenterAvailability()
+          && !nodeDc.equals(localDc)
+          && context.storage instanceof IDistributedStorage) {
```

Comment:
is there any reason that `datacenterAvailability` would be `EACH` or `LOCAL` when `IDistributedStorage` was not used?

if so, it would be better to place a `Precondition.check..(..)` in `ReaperApplication.initializeStorage(..)`?

Reply:
per f2f discussion w/ @adejanovski `EACH` and `LOCAL` should be prevented, as described above.

Reply:
I have allowed `LOCAL` as it could still be a valid usage (although not safe) of non distributed storage backends in multi DC setups.
`EACH` would be the only case where non distributed storage is not suitable.
Works for you @michaelsembwever ?

Reply:
> Works for you @michaelsembwever ?

WFM